core/config/toml/types.go

```diff
@@ -1456,9 +1456,11 @@ func (eerl *EngineExecutionRateLimit) setFrom(f *EngineExecutionRateLimit) {
 }
 
 type ExternalRegistry struct {
-	Address   *string
-	NetworkID *string
-	ChainID   *string
+	Address                *string
+	NetworkID              *string
+	ChainID                *string
+	GlobalEngineCountLimit *int32
+	WorkflowsPerOwnerLimit *int32
```

Comment:
Can we put these two fields in a new top-level field called Workflows? I think putting it in ExternalRegistry isn't quite right (that refers to the CapabilitiesRegistry btw, not the WorklfowRegistry)